Dockerfile.ci

```diff
@@ -53,7 +53,8 @@ pip install --no-cache-dir --no-build-isolation --extra-index-url https://pypi.n
 ".[all]"
 
 # Megatron Core installation
-git clone https://github.com/NVIDIA/Megatron-LM.git && \
+# TODO: revert original repo
+git clone https://github.com/mikolajblaz/Megatron-LM.git && \
```

Comment:
can we wait until that changes merged to mcore?

Reply:
yes

Reply:
updated to public commit